test/Microsoft.AspNet.Mvc.FunctionalTests/TestHelper.cs

```diff
@@ -131,6 +131,26 @@ private static IServiceProvider BuildFallbackServiceProvider(IEnumerable<IServic
             return sc.BuildServiceProvider();
         }
 
+        public static string NormalizeExpectedUrl(this string url)
```

Comment:
Would rather this be a static method instead of an extension. That way it doesn't add unnecessary extension methods to all string classes in the tests.